### ISSUE TITLE
Unable to build on Anrdoid

### ISSUE THREAD

**Issue by zzz08900** [guhungry/react-native-photo-manipulator#806]
react-native: 0.72.12
node.js: 18.20.2
react-native-photo-manipulator: 1.4.0

Related error log:
Could not determine the dependencies of task ':react-native-photo-manipulator:bundleLibCompileToJarDebug'.
> Could not determine the dependencies of null.
   > Could not create task ':react-native-photo-manipulator:compileDebugKotlin'.
      > Cannot use @TaskAction annotation on method AbstractKotlinCompile.execute() because interface org.gradle.api.tasks.incremental.IncrementalTaskInputs is not a valid parameter to an action method.

It works all right on iOS, with the same project and node.js version.
I also tried 1.3.2 and was hit by a similiar message, saying that it `Could not determine the dependencies of task ':react-native-photo-manipulator:xxxxxxx'.`

**Comment by zzz08900** [comment]
Ok here's what I found:
1. The `build.gradle` from `example` folder locked kotlinVersion to 1.9.x, but the package's default value is 1.5.x, maybe we should update the package's default value?
2. The `example` project was built with react-native 0.73 and there's was a change in the constructor of class `ReactModuleInfo` so the package does not build with RN 0.72. In RN 0.73 the constructor param `boolean hasConstants` was removed. To be honest I have no idea what this param means and I just put in a `false` within `RNPhotoManipulatorPackage.java` and everything seems to be working order.

So here's what I think:
1. We really should update  the package's default kotlinVersion, using 1.5.30 when the package was supposed to be building with 1.9.x will likely cause chaos in, well, everywhere
2. I don't see a way of supporting RN 0.72 and 0.73 at the same time without creating another `RNPhotoManipulatorPackage.java` file so maybe we should update the docs saying this package only works with RN 0.73+?
